File: spec/code_samples/Node_unirest/list@cleaned@{list_name}/delete.js
var unirest = require("unirest");

var req = unirest("DELETE", "https://cleanse.studio/api/v1/list/cleaned/%7Blist_name%7D");

req.query({
  "username": "SOME_STRING_VALUE"
});

req.headers({
  "content-type": "application/json"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Python_python3/list@cleaned@{list_name}/delete.py
import http.client

conn = http.client.HTTPSConnection("cleanse.studio")

headers = { 'content-type': "application/json" }

conn.request("DELETE", "/api/v1/list/cleaned/%7Blist_name%7D?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))